flink-cdc-cli/src/main/java/com/ververica/cdc/cli/parser/YamlPipelineDefinitionParser.java

```diff
@@ -50,9 +49,9 @@ public class YamlPipelineDefinitionParser implements PipelineDefinitionParser {
     private static final String NAME_KEY = "name";
 
     // Route keys
-    private static final String MATCHER_KEY = "matcher";
-    private static final String REPLACE_KEY = "replace";
-    private static final String DESCRIPTION_KEY = "description";
+    private static final String ROUTE_SOURCE_TABLE_KEY = "sourceTable";
```

Comment:
Will `sourceTable` be better?

Reply:
The best naming in my mind is:
`sourceTable` -> `matcher`: this gives a hint to user that a regex is required here
`sinkTable` -> `replaceBy`: this will be used to replace the original table ID. 
which is like the replacing functionality in text editor.

Reply:
Let's keep the simple name as we designed, source/sink concepts is better as flink users are familiar with them